developer-guides/rest-api/authentication/logout/README.md

```diff
@@ -11,7 +11,7 @@ Invalidate your REST API authentication token.
 ```bash
 curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
      -H "X-User-Id: aobEdbYhXfu5hkeqG" \
-     http://localhost:3000/api/v1/logout
+     -X POST http://localhost:3000/api/v1/logout
```

Comment:
See: [https://github.com/kahmali/meteor-restivus/issues/100](https://github.com/kahmali/meteor-restivus/issues/100) and [https://github.com/RocketChat/docs/pull/1245/commits/e0d12c1c5b62d56adaa5d6752828b2e08e3010e2#diff-102cded5718fc2504f73d10f9fe05e91R7](https://github.com/RocketChat/docs/pull/1245/commits/e0d12c1c5b62d56adaa5d6752828b2e08e3010e2#diff-102cded5718fc2504f73d10f9fe05e91R7)